### DISCUSSION TITLE
There is no Vercel account linked with this GitHub account

### DISCUSSION THREAD

**Discussion by dormouseroared** [vercel/community#920] [Help] [answered]
hi

First time in ages, but signing into vercel with github gets this response.
I just updated a repo that gets built by vercel and that is working fine.
No real clues on searching so looking for some help.

Thanks!

**Comment by swarnava** [comment, marked as answer]
Hi,

Can you try login using your email and let us know how it goes?

**Reply by dormouseroared** [reply]
Hi

Thanks!

First time, waiting to be redirected to the dashboard...
Second time, success.
Does this reconnect my github and vercel accounts, or do I get in this way using email from now on.

Cheers!

**Reply by swarnava** [reply]
You can check if your Git account is connected or not here: https://vercel.com/account/login-connections. Once it's connected, you can use either of these methods to login.

**Reply by dormouseroared** [reply]
hi

Many, many thanks!
